pandas/tests/io/json/test_pandas.py

```diff
@@ -126,19 +118,19 @@ def test_frame_non_unique_columns_raises(self, orient):
         with pytest.raises(ValueError, match=msg):
             df.to_json(orient=orient)
 
-    def test_frame_default_orient(self):
-        assert self.frame.to_json() == self.frame.to_json(orient="columns")
+    def test_frame_default_orient(self, float_frame):
+        assert float_frame.to_json() == float_frame.to_json(orient="columns")
```

Comment:
Note that `float_frame` and `self.frame` aren't 100% equivalent because the former has the columns labeled ABCD whereas the latter didn't provide labels. I don't think that distinction matters for these tests though